### PR TITLE
dnn: add colorization.cpp

### PR DESCRIPTION
**Pull request by berak**
add a cpp counterpart for the colorization.py sample, also fix some links in the python script

### CODE REVIEW

File: samples/dnn/colorization.cpp [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    -90., -80., -70., -60., -50., -40., -30., -20., -10., 0., 10., 20., 30., 40., 50., 60., 70., 80., -110., -100., -90., -80., -70.,
+    -60., -50., -40., -30., -20., -10., 0., 10., 20., 30., 40., 50., 60., 70., -110., -100., -90., -80., -70., -60., -50., -40., -30.,
+    -20., -10., 0., 10., 20., 30., 40., 50., 60., 70., -90., -80., -70., -60., -50., -40., -30., -20., -10., 0.
+};
+
+
+int main(int argc, char **argv)
+{
+    CommandLineParser parser(argc, argv,
+        "{ help           | false | Sample app for loading googlenet model }"
```
Comment (sturkmen72) [inline samples/dnn/colorization.cpp]: something like "print usage" or "print help message"

**Comment by sturkmen72** [comment]
i did not find `colorization_release_v2.prototxt`
instead i used [colorization_deploy_v2.prototxt](https://raw.githubusercontent.com/richzhang/colorization/master/colorization/models/colorization_deploy_v2.prototxt) and [colorization_release_v2.caffemodel](
https://people.eecs.berkeley.edu/~rich.zhang/projects/2016_colorization/files/demo_v2/colorization_release_v2.caffemodel)
maybe a bit more info about needed files will be better

File: samples/dnn/colorization.cpp (hunk 1 above)
Comment (berak) [reply samples/dnn/colorization.cpp]: indeed. leftover from the googlenet sample.

**Comment by vpisarev** [comment]
@dkurt, can you please review it and update the comment in the beginning of the script with proper paths to the models?

**Pull request merged**